addons/ai/functions/fnc_unpackStaticWeapon.sqf

```diff
@@ -0,0 +1,50 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Unpacks a static weapon from units' backpacks.
+ *
+ * Arguments:
+ * 0: Gunner <OBJECT>
+ * 1: Assistant <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_gunner, _assistant] call zen_ai_fnc_unpackStaticWeapon
+ *
+ * Public: No
+ */
+
+params ["_gunner", "_assistant", ["_targetPos", [], [[]], 3]];
+if !(local _gunner) exitWith {_this remoteExecCall [FUNC(unpackStaticWeapon), _gunner]};
+
+_gunner setVariable [QGVAR(unpackStaticWeaponTargetPos), _targetPos];
+
+_gunner addEventHandler ["WeaponAssembled", {
+    params ["_gunner", "_weapon"];
+
+    _gunner removeEventHandler ["WeaponAssembled", _thisEventHandler];
+
+    _weapon setVectorUp surfaceNormal position _weapon;
```

Comment:
Why? Is the weapon not placed correctly when assembled?

Reply:
Sometimes! I can compile a list if needed.

Reply:
This issue demo'd with base-game function. https://youtu.be/D0nBZD2xQSk